README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
“smart” means inserting them or is used as contrast to “dumb” (typewriter) quotes.

in fact they’re *real* quotes and typewriter quotes were originally a hack because typewriters were technically limited.

today, [the typographic standard is to use curly quotes for quoted phrases in text](http://practicaltypography.com/straight-and-curly-quotes.html) (citations, …). as a convention, backticks often get used for code, inside and out of markdown. and finally, typewriter quotes mean *string*. i.e. a piece of textual data.

so i hope you can shed your conditioning that the mistakes of bad tools imprinted on you, and come to appreciate good typography in time.

> BTW:
> 
> ```R
> >>> "# “ir” is"
> "# ir is"
> ```
> 
> [In the notebook]

sure, because you pasted it as string. string encoding in R on windows is fucked up, i know. if you’d have pasted it as comment, nothing would have happened because comments don’t get interpreted.